packages/vscode-extension/package.nls.cs.json

```diff
@@ -1,607 +1,607 @@
-{

```

Comment:
@HuihuiWu-Microsoft It seems the content is not changed, may be caused by format change. Is it ok to merge?

Reply:
It seems I mistakenly replaced the ELS. How about I update the ELS back to LF in the next PR if it's ok to merge first

Reply:
Nice catch! Better keep the files unchanged. If the changes persist for every commit, you can double-check the End of Line Sequence setting in vsc.

Reply:
@huimiu Yes, please change it back in next pull request.

Reply:
Thanks!